tower-http/src/sensitive_headers.rs

```diff
@@ -0,0 +1,376 @@
+//! Middlewares that mark headers as [sensitive].
```

Comment:
Would it maybe make sense to break out the request/response ones into their own module? Might make it a bit easier to read.